sites/stephaniewalter.design.md

```diff
@@ -0,0 +1,8 @@
+---
+title: 'Stéphanie Walter'
+url: 'https://stephaniewalter.design/'
+tags: ['UX Research', 'UX Design' 'Inclusive Design','Accessibility','Speaker', 'Writer', 'Teacher', 'Newsletter']
```

Comment:
The é didn't seem to be a problem, but we're missin' a comma in this list! Thanks for opening the PR :)